services/web/server/src/simcore_service_webserver/resource_usage/_client.py

```diff
@@ -139,6 +141,7 @@ async def sum_total_available_credits_in_the_wallet(
     )
     with handle_client_exceptions(app) as session:
         async with session.post(url) as response:
+            response.raise_for_status()
```

Comment:
TIP: if you are raising everywhere, perhaps you just activate it at the session level. i.e `aiohttp.ClientSession(...raise_for_status=True)`. SEE  https://docs.aiohttp.org/en/stable/client_reference.html#client-session